libs/core/langchain_core/messages/tool.py

```diff
@@ -39,6 +55,16 @@ class ToolMessage(BaseMessage):
     type: Literal["tool"] = "tool"
     """The type of the message (used for serialization). Defaults to "tool"."""
 
+    raw_output: Dict[str, Any] = Field(default_factory=dict, exclude=True)
+    """The raw output of the tool.
+    
+    **Not part of the payload sent to the model.** Should only be specified if it is 
+    different from the message content, i.e. if only a subset of the full tool output
```

Comment:
(nit) does raw_output always contain content or could content be derived from raw output?